README.rst

```diff
@@ -84,3 +84,70 @@ Store
 ^^^^^
 
 Store will be able to have caching, real time data update, network capabilities (provided with component embedded in StoreLoader) to manage data... it will be the main piece to fetch and share data accross UI components. It uses native Qt binding (see Search.qml as an example)
+
+
+Feature Projects
+^^^^^
+
+Section Project parameters, example :
+
+{
```

Comment:
doesn't look as expected.
you can add something similar to
.. code:: bash

and indent all projects config with 4 spaces

you can use something like http://rst.ninjs.org/ to check the rst file.